src/openassetio-core/src/pluginSystem/CppPluginSystemManagerImplementationFactory.cpp

```diff
@@ -46,8 +51,7 @@ CppPluginSystemManagerImplementationFactory::CppPluginSystemManagerImplementatio
     : CppPluginSystemManagerImplementationFactory{
           // getenv returns nullptr if var not set, which cannot be
           // used to construct a std::string.
-          // False positive in clang-tidy-12:
-          // NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg)
+          // NOLINTNEXTLINE(*-suspicious-stringview-data-usage)
           [paths = std::getenv(kPluginEnvVar.data())] { return paths ? paths : ""; }(),
```

Comment:
I recall writing something, but then can't see it now for this. Must've lost it in a refresh.

The *-suspicious-stringview-data-usage is for the use of `std::getenv(kPluginEnvVar.data())`.

There is a std::getenv_s (https://en.cppreference.com/w/c/program/getenv) which should get around that, but at the cost of more code.

This is a suggestion more than anything, if it appeals.

Reply:
Drat.
end of line.